src/sentry/middleware/integrations/classifications.py

```diff
@@ -131,7 +135,15 @@ def get_response(self, request: HttpRequest) -> HttpResponseBase:
             request=request,
             response_handler=self.response_handler,
         )
-        response = parser.get_response()
+        try:
+            response = parser.get_response()
+        except (Integration.DoesNotExist, OrganizationIntegration.DoesNotExist):
+            metrics.incr(
+                f"hybrid_cloud.integration_control.integration.{parser.provider}",
+                tags={"url_name": parser.match.url_name, "status_code": 404},
+            )
+            return Response(status=status.HTTP_404_NOT_FOUND)
```

Comment:
This is a much better catch all than the 400 responses I've been adding to each parser 😅 I'll push a patch to fallback to this instead.